### PR TITLE
Bump PHP Assoc map

### PR DESCRIPTION
**Pull request by nicklayb**
Should address issue #380

The library was updated to use a parser/lexer rather than a tokenizer but was never updated in accent. Then another issue caused special characters to be casted as binary rather that strings.

### CODE REVIEW

File: test/langue/laravel_php/expectation_test.exs [hunk 1]
```diff
@@ -9,8 +9,9 @@ defmodule LangueTest.Formatter.LaravelPhp.Expectation do
     def render do
       """
       <?php
+
       return [
-        'required'=>'Le champ :attribute est obligatoire.',
+        'required' => 'Le champ :attribute est obligatoire.',
```
Comment (nicklayb) [inline test/langue/laravel_php/expectation_test.exs:14]: This is just to make sure that spaced arrows are also supported

**Comment by nicklayb** [comment]
I managed to import the following

```
<?php

return [
  "key" => "value",
  'key' => 'value',
  'something' => [
    'key' => "Value avec êàé"
  ],
  "somehting_else" => "Allo"
];
```

Without any issues  which includes double and single quote as well as special characters

**Review by simonprev — APPROVED** [review]
Oh awesome! Thanks Nic!

**Comment by nicklayb** [comment]
@simonprev Wait before merging it, @guyaumetremblay identified an issue with trailing commas. I'm updating the lib now, it shouldn't be too long

**Comment by guyaumetremblay** [comment]
@simonprev Hi Simon! Do you know when a new tag will be available with this fix please?

**Pull request merged**